SU2_CFD/include/variable_structure.hpp

```diff
@@ -2482,6 +2482,39 @@ class CWaveVariable : public CVariable {
   
 };
 
+/*!
+ * \class CHeatFVMVariable
+ * \brief Main class for defining the variables of the finite-volume heat equation solver.
+ * \author O. Burghardt
+ * \version 5.0.0 "Raven"
```

Comment:
Same, version number. Same comments apply to CVariable - we can eventually make these the only Heat classes.